.jshintrc

```diff
@@ -0,0 +1,9 @@
+{
+  "node": true,
+  "curly": true,
+  "latedef": true,
+  "quotmark": true,
+  "undef": true,
+  "unused": true,
+  "trailing": true
+}
```

Comment:
This is different from the settings we use in other repos?
ref: https://github.com/pelias/api/blob/master/.jshintrc